api_docs/incoming-webhooks-overview.md

```diff
@@ -155,3 +155,32 @@ below are for a webhook named `MyWebHook`.
   testing with live data from the service you're integrating and can help you
   spot why something isn't working or if the service is using custom HTTP
   headers.
+
+## URLs
+
+The base URL for an incoming webhook integration bot is
+`{{ api_url }}/v1/external/integration_name?api_key=abcdefgh` where
+`integration_name` is the name of the specific webhook integration and
+`api_key` is the API key of the bot created by the user for the
+integration.
+
+The list of existing webhook integrations can be found in
+`zerver/lib/integrations.py` (see `WEBHOOK_INTEGRATIONS`) or by browsing
```

Comment:
```suggestion
`zerver/lib/integrations.py` (at `WEBHOOK_INTEGRATIONS`) or by browsing
```

or perhaps "see `WEBHOOK_INTEGRATIONS` there"

My first reading was that "see `WEBHOOK_INTEGRATIONS`" was a reference meant to further elaborate on the meaning of "`zerver/lib/integrations.py`", so this is just to disambiguate from that.